source/Drawers.js

```diff
@@ -348,12 +349,42 @@
 		/**
 		* @private
 		*/
+		hideDrawers: function(except) {
+			var d$ = this.$.drawers.getControls();
+			for (var i=0; i<d$.length; i++) {
+				d$[i].setShowing(d$[i] === except);
+			}
+		},
+
+		/**
+		* @private
+		*/
+		animate: function(height) {
+			enyo.dom.transform(this.$.drawers, {translateY: height + 'px'});
+			enyo.dom.transform(this.$.client, {translateY: height + 'px'});
+		},
+
+		/**
+		* @private
+		*/
 		drawerActivated: function (sender, ev) {
-			if (ev.originator instanceof moon.Drawer) {
-				this.updateActivator(true);
-				// Hide client when fullscreen drawer is open so it is not focusable
-				if (ev.originator.getOpen()) {
-					this.$.client.hide();
+			if (moon.config.accelerate) {
+				if (ev.originator instanceof moon.Drawer) {
+					this.updateActivator(true);
+					// Hide() is expensive. Use spotlightDisabled feature instead
+					if (ev.originator.getOpen()) {
+						this.$.client.spotlightDisabled = true;
+					}
+					this.hideDrawers(ev.originator);
+					this.animate(ev.height);
+				}
+			} else {
+				if (ev.originator instanceof moon.Drawer) {
+					this.updateActivator(true);
+					// Hide client when fullscreen drawer is open so it is not focusable
+					if (ev.originator.getOpen()) {
```

Comment:
Hi @KunmyonChoi, with this conditional when we have set `moon.config.accelerate` to `false`, any drawer that is hidden initially will not be shown. You can see an example of this in the `DrawerSample` when trying to open the partial drawer.

cc @yeram